src/middlewares/ClientErrorHandler.ts

```diff
@@ -14,9 +14,11 @@ import HttpException from "../exceptions/HttpException";
  * Handle exceptions thrown from controllers/services.
  */
 class ClientErrorHandler implements IMiddleware {
-  // eslint-disable-next-line max-params
```

Comment:
Based on what I commented on eslint config file, let's leave this as an exception